feature-libs/asm/root/services/asm-auth.service.ts

```diff
@@ -39,6 +40,7 @@ export class AsmAuthService extends AuthService {
     protected authRedirectService: AuthRedirectService,
     protected globalMessageService: GlobalMessageService,
     protected routingService: RoutingService,
+    protected winRef: WindowRef,
```

Comment:
Same situation, it's related with: https://github.com/SAP/spartacus/pull/17232#discussion_r1159477011

Reply:
Changed